examples/music-licensing-challenge/src/app/api/scenes.py

```
@@ -0,0 +1,24 @@
+from typing import List, Optional
+from ..db.database import get_db
+from ..models.scene import Scene
+from ..schemas.scenes import SceneWithAllData
+from ..repository.scenes import SceneRepository
+
+router = APIRouter()
+
+
+@router.get("/", response_model=List[SceneWithAllData] | SceneWithAllData)
+def read_scenes(
+    id: Optional[str] = None,
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)